convert_hf_to_gguf.py

```diff
@@ -9829,6 +9832,46 @@ def prepare_tensors(self):
                 raise ValueError(f"Unprocessed experts: {experts}")
 
 
+@ModelBase.register("ModernBertModel", "ModernBertForMaskedLM", "ModernBertForSequenceClassification")
+class ModernBertModel(BertModel):
+    model_arch = gguf.MODEL_ARCH.MODERN_BERT
+
+    def set_vocab(self):
+        self.gguf_writer.add_add_bos_token(True)
+        self.gguf_writer.add_add_eos_token(True)
+        self.gguf_writer.add_add_sep_token(True)
+        self._set_vocab_gpt2()
+
+    def set_gguf_parameters(self):
+        super().set_gguf_parameters()
+        self.gguf_writer.add_dense_every_n_layers(self.hparams["global_attn_every_n_layers"])
+        self.gguf_writer.add_sliding_window(self.hparams["local_attention"])
+        self.gguf_writer.add_rope_freq_base(self.hparams["global_rope_theta"])
```

Comment:
It looks like something may have changed in `transformers` v5. When I try to run `convert_hf_to_gguf.py` it's erroring out with this as a missing key. It looks like that is no longer a standard key in the config object: https://github.com/huggingface/transformers/blob/main/src/transformers/models/modernbert/configuration_modernbert.py#L38